src/netappfiles-preview/HISTORY.rst

```diff
@@ -2,6 +2,10 @@
 
 Release History
 ===============
+0.5.2
```

Comment:
version here should be consistent with setup.py, which is 1.0.0b2

Reply:
Fixed